src/Widgets/ImageWidget/DimensionsEditor.tsx

```diff
@@ -8,23 +8,33 @@ export function DimensionsEditor({ widget }: { widget: ImageWidgetInstance }) {
   const { theme } = uiContext() || { theme: null }
   if (!theme) return null
 
+  const readOnly = !canEdit(widget.obj()) || isComparisonActive()
+  const muteObjectFit = !widget.get('height')
+
   return (
     <div
       className={`dimensions-editor scrivito_detail_content scrivito_${theme}`}
     >
       <div className="row">
         <div className="col-auto">
-          <DimensionEditor widget={widget} attribute="width" label="Width" />
+          <DimensionEditor
+            widget={widget}
+            attribute="width"
+            label="Width"
+            readOnly={readOnly}
+          />
         </div>
         <div className="col-auto">
           <DimensionEditor
             widget={widget}
             attribute="height"
             label="Height"
             pxOnly
+            readOnly={readOnly}
           />
         </div>
       </div>
+      <ObjectFit widget={widget} readOnly={readOnly || muteObjectFit} />
```

Comment:
Please work on this in a follow up:

* I think it would be better, if `ObjectFit` always shows "Contains", regardless of the `objectFit` value, if no height is provided
* The widget rednering component should not add a `object-fit` property, if `height` is empty.

Reply:
> I think it would be better, if ObjectFit always shows "Contains", regardless of the objectFit value, if no height is provided

This would be inconsistent with other conditionally disabled controls, e.g.:
<img width="490" alt="image" src="https://github.com/user-attachments/assets/a4b9469a-a935-4742-8a6a-b177921115be" />

Since without height the behavior is the same regardless of the object-fit setting, it has even less potential for confusion.

However, with object-fit being last in the tab and having no meaning w/o height, how about hiding it completely in this case?